meta-balena-common/recipes-containers/hostapp-update/files/hostapp-update

```diff
@@ -37,19 +45,244 @@ run_current_hooks_and_recover () {
 	exit 1
 }
 
+# Test if a version is greater than another
+version_gt() {
+	test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" != "$1"
+}
+
+# Dashboard progress helper
+progress() {
+	percentage=$1
+	message=$2
+	resin-device-progress --percentage "${percentage}" --state "${message}" > /dev/null || true
+}
```

Comment:
It seems progress() is only used for the supevisor update and not the hostapp update? I expect the supervisor is already reporting progress to the cloud - I would remove this from the hostOS and centralize progress reports in the supervisor.

Reply:
As Shaun said in Zulip, indeed the supervisor should report progress for the OS update. This here is progress of the supervisor update which is done in the overall OS update. I can remove this altogether from here if desired